app/src/main/java/co/tinode/tindroid/services/FBaseMessagingService.java

```diff
@@ -248,6 +248,14 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
                         Log.d(TAG, "No new data. Skipping notification.");
                         return;
                     }
+
+                    // If it's a video call, start it.
+                    if (UiUtils.isVideoCallMime(data.get("mime"))) {
```

Comment:
This combination of UiUtils.isVideoCallMime(data.get("mime")) & if (data.get("replace") == null)  is used in a few places, maybe extract it into a method.